app/src/main/res/values/strings.xml

```diff
@@ -18,330 +18,334 @@
-    <string name="education_unlock_summary">Enter a password and/or a key file to unlock your database.\n\nRemember to save a copy of your .kdbx file in a safe place after each modification.</string>
-    <string name="education_read_only_title">Enable read-only</string>
-    <string name="education_read_only_summary">Change the opening mode for the session.\n\nIn read-only mode, you prevent unintended changes to the database.\n\nIn write mode, you can add, delete, or modify all the elements as you want.</string>
+    <string name="education_unlock_summary">Enter the password and/or saved fingerprint to unlock your database.\n\nBackup your database file in a safe place after each change.</string>
+    <string name="education_read_only_title">Write protect your database</string>
+    <string name="education_read_only_summary">Change opening mode for the session.\n\n\"Write-protected\" prevents unintended changes to the database.\n\"Modifiable\" lets you add, delete, or modify all elements.</string>
     <string name="education_field_copy_title">Copy a field</string>
-    <string name="education_field_copy_summary">Copy a field easily to paste it where you want\n\nYou can use several forms filling methods. Use the one you prefer.</string>
+    <string name="education_field_copy_summary">Copied fields can be pasted anywhere.\n\nUse the form filling method you prefer.</string>
     <string name="education_lock_title">Lock the database</string>
-    <string name="education_lock_summary">Lock your database quickly, you can set up the app to lock it after a while, and when the screen goes off.</string>
-    <string name="education_sort_title">Sort items</string>
-    <string name="education_sort_summary">Sort entries and groups according to specific parameters.</string>
+    <string name="education_lock_summary">Lock your database quickly, you can set up the app to lock it after a while, and when the screen turns off.</string>
+    <string name="education_sort_title">Item sorting</string>
+    <string name="education_sort_summary">Choose how entries and groups are sorted.</string>
     <string name="education_donation_title">Participate</string>
-    <string name="education_donation_summary">Participate to help increase the stability, security and in adding more features.</string>
+    <string name="education_donation_summary">Help increase the stability, security and in adding more features.</string>
 
-    <string name="html_text_ad_free">Unlike many password management apps, this one is &lt;strong&gt;ad-free&lt;/strong&gt;, &lt;strong&gt;copylefted libre software&lt;/strong&gt; and does not recover personal data on its servers, even in its free version.</string>
+    <string name="html_text_ad_free">Unlike many password management apps, this one is &lt;strong&gt;ad-free&lt;/strong&gt;, &lt;strong&gt;copylefted libre software&lt;/strong&gt; and does not collect personal data on its servers, even in its gratis version.</string>
```

Comment:
even in its "Free" version. There is no gratis version

Reply:
Then it makes no sense to me.